src/modules/completed.js

```diff
@@ -0,0 +1,8 @@
+export default function () {
+  const tasksCompleted = document.querySelectorAll('.checked');
+  const arrCompleted = [...tasksCompleted];
+  for (let i = 0; i < arrCompleted.length; i += 1) {
+    arrCompleted[i].parentElement.remove();
+    arrCompleted[i].nextElementSibling.remove();
+  }
+}
```

Comment:
- [ ] I think  The complete is not toggling in the local storage from false to true, so kindly fix that since it is from the requirements.

- [ ] After refreshing I can't see any tasks displayed in the list so kindly fix it🔥